### PR TITLE
feat(#2w79jf_#4p6r5j): Initialize Navbar component and add Icons library

### PR DESCRIPTION
**Pull request by Kevinjhlin**
Add the Navbar component as well as an Icon pack but the icons do not lead to anywhere until the other components are done as well.

### CODE REVIEW

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './styles.scss';
+import {Nav, NavItem, NavLink} from "shards-react";
+import {FaBars, FaHome, FaMoneyBillWave} from 'react-icons/fa';
+// FaBars FaHome FaMoneyBillWave
```
Comment (alphakennyn) [inline src/components/navbar/Navbar.jsx]: Remove pls

File: src/components/navbar/Navbar.test.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Navbar from './Navbar.jsx';
+
+describe('LoginPage', () => {
```
Comment (alphakennyn) [inline src/components/navbar/Navbar.test.jsx]: Update the names please

**Review by alphakennyn — CHANGES_REQUESTED** [review]

**Comment by alphakennyn** [comment]
Dont forget to relace the `Header` tag with your navbar in the `app.js`

File: src/components/navbar/Navbar.jsx (hunk 1 above)
Comment (Kevinjhlin) [reply src/components/navbar/Navbar.jsx]: @alphakennyn but if I remove it then wouldn't it compile incorrectly as the icons depend on it? What should I use instead then? It's place holders should I remove them when we have the assets or do we already have them?

Comment (alphakennyn) [reply src/components/navbar/Navbar.jsx]: I was referring to `line 5` which is commented-out code. Keep our code clean

**Review by alphakennyn — APPROVED** [review]

**Pull request merged**